cmd/vic-machine/common/certificate.go

```diff
@@ -55,6 +55,7 @@ type CertFactory struct {
 	NoTLSverify  bool
 	KeyPEM       []byte
 	CertPEM      []byte
+	NoSaveToDisk bool
```

Comment:
We should consider calling this `SaveToDisk` since `if c.SaveToDisk`is easier to understand than `if !c.NoSaveToDisk` for all the checks below. Also, since booleans are false by default `c.Certs.NoSaveToDisk = true` wouldn't be needed either.

Reply:
Normally, I would prefer to avoid "negatives" in boolean names. In this case, I think we want to default to saving the files to disk (as the CLI is still the primary way of interacting with this package) and we have exactly one code path where we don't want to save to disk (the create API handler).

I _think_ it's helpful to word the name such that the zero value is the one we want by default, even if it means we have to read double negatives in a few places.

Reply:
Makes sense 👍